openfecwebapp/templates/partials/legal-search-results-advisory-opinion.html

```diff
@@ -1,8 +1,18 @@
-<tr>
-  <td><a title="{{ advisory_opinion.description }}" href="http://saos.fec.gov/aodocs/{{ advisory_opinion.no }}.pdf#search={{ query }}">{{ advisory_opinion.no }}</a></td>
+<tr class="legal-search-result">
+  <td><a title="{{ advisory_opinion.description }}" href="{{ advisory_opinion.pdf_url }}">AO {{ advisory_opinion.no }}</a></td>
   <td>{{ advisory_opinion.name }}</td>
   <td>
     <strong>{{ advisory_opinion.summary }}</strong>
-    <div class="t-serif" title="{{advisory_opinion.text}}">{{ advisory_opinion.text|truncate }}</div>
+    <div class="t-serif legal-search-result__hit">
+      &hellip;
+      {% for highlight in advisory_opinion.highlights %}
+        {{ highlight|safe }} &hellip;
+      {% endfor %}
+    </div>
+    {% if legal_include_display_all %}
+    <div class="legal-search-result__display-all">
+      <a href="/legal/advisory-opinions?search={{ query }}">Display all hits</a>
```

Comment:
Yeah, I think I'd vote for removing it (even just removing the `href` so you can still see how it looks with that link there), but don't feel too strongly.